packages/nc-gui-v2/components/virtual-cell/components/ListChildItems.vue

```diff
@@ -1,14 +1,14 @@
 <script lang="ts" setup>
 import { Modal } from 'ant-design-vue'
 import type { ColumnType } from 'nocodb-sdk'
-import { useLTARStoreOrThrow, useSmartsheetRowStoreOrThrow, useVModel, watch } from '#imports'
+import { useLTARStoreOrThrow, useSmartsheetRowStoreOrThrow, useVModel, watch, computed } from '#imports'
 import { ColumnInj, IsFormInj } from '~/context'
 
 const props = defineProps<{ modelValue?: boolean }>()
 const emit = defineEmits(['update:modelValue', 'attachRecord'])
 
 const vModel = useVModel(props, 'modelValue', emit)
-const isForm = inject(IsFormInj, false)
+const isForm = ref(inject(IsFormInj, false))
```

Comment:
why not `inject(isForm, ref(false))`?
or just make the injection an actual ref?